.github/workflows/packages-filebeat.yml

```
@@ -0,0 +1,58 @@
+name: Packages - Build filebeat package
+on:
+  workflow_dispatch:
+    inputs:
+      revision:
+        description: 'Revision used to naming Filebeat package wazuh-filebeat-X.X.tar.gz'
```

I think it would be more correct to use `version`, rather than `revision`, as _revision_ would be in the case after _version_. Example: `wazuh-filebeat-X.X`**`-X`**
```suggestion
      version:
        description: 'Version used to naming Filebeat package wazuh-filebeat-X.X.tar.gz'
```